_posts/databases/postgresql/TimescaleDB/2000-01-01-intro.md

```diff
@@ -0,0 +1,20 @@
+---
+title: TimescaleDB PostgreSQL Extensions
+nav: Extensions
+modified_at: 2022-01-21 00:00:00
+tags: timescale databases postgresql extensions
+index: 1
+---
+
+TimescaleDB is a PostgreSQL extension for the support of the time-series data.
+The version available on Scalingo is the Open-source edition, which is different
+than the Community edition. You can check the comparison between both edition,
```

Comment:
By being an external reader, I don't understand what you are talking about with this "Community edition". It seems it comes out of nowhere. Maybe it's less odd to say that the list of TimescaleDB functions introduced by enabling TimescaleDB on a PostgreSQL database is here (https://docs.timescale.com/api/latest/). From this list, the functions labelled as `Community` are not available on Scalingo.